components/Avatar/avatar.css

```diff
@@ -0,0 +1,31 @@
+.avatar {
+  object-fit: cover;
+  width: 45px;
+  height: 45px;
+  margin: 0 1rem;
```

Comment:
@vinc3nati Try to stick with only one units either REM or PX.

Reply:
I need margins to change along with the root and not the dimensions of the element so..